src/Resources/views/CRUD/Association/edit_one_to_many_inline_table.html.twig

```diff
@@ -26,8 +26,7 @@ file that was distributed with this source code.
                         {% if nested_field.vars.translation_domain is same as(false) %}
                             {{ nested_field.vars.label }}
                         {% else %}
-                            {% set translationDomain = nested_field.vars.translation_domain|default(nested_field.vars['sonata_admin'].admin.translationDomain) %}
```

Comment:
Is there some way that `nested_field.vars['sonata_admin'].admin.translationDomain` was actually used a the translation domain?

if `nested_field.vars.translation_domain` is `false` then its the above if block anyway?

Reply:
Ah wait. It can be `null`... changing it

Reply:
Previously the https://github.com/sonata-project/SonataAdminBundle/pull/6401 PR, it was:
```
nested_field.vars['sonata_admin'].admin.translationDomain|default(nested_field.vars.translation_domain)
```

Which mean it was used. And when it wasn't defined (like in your case) `nested_field.vars.translation_domain` was used instead.

The PR (with the bug), swap the default value.

I would say that if `nested_field.vars.translation_domain` is `null` it will fallback to `nested_field.vars['sonata_admin'].admin.translationDomain`, which is not a bad thing.

But if both `nested_field.vars.translation_domain` is `null` and `nested_field.vars['sonata_admin'].admin.translationDomain` is not defined ; what should we do ?

Reply:
if both are not available then we use `null` as the domain. The Symfony translator should handle it properly and use the default domain I think?

Reply:
Seems like the right solution indeed.